cmd/queue/main.go

```diff
@@ -281,7 +281,7 @@ func main() {
 		if queueDepth < 10 {
 			queueDepth = 10
 		}
-		breaker = queue.NewBreaker(int32(queueDepth), int32(containerConcurrency))
+		breaker = queue.NewBreaker(int32(queueDepth), int32(containerConcurrency), int32(containerConcurrency))
```

Comment:
Nit: can you enumerate these parameters with names by either making them unique types or using a struct please?  Previously I have gotten these confused.  And now there are three int32's.

E.g.
```golang
type Depth int32
type Max int32
type Initial int32
breaker = queue.NewBreaker(queue.Depth(queueDepth), queue.Max(containerConcurrency), queue.Initial(containerConcurrency))
```
or
```golang
type BreakerParams struct {
    Depth    int32
    Max      int32
    Initial  int32
}
breaker = queue.NewBreaker(BreakerParams{Depth: queueDepth, Max: containerConcurrency, Initial: containerConcurrency})
```

Reply:
I just left them as they are for the moment, the Breaker could change in the next PRs anyway, will defer until then